modules/integration/tests-ui/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright 2009 WSO2, Inc. (http://wso2.com)
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/f021f86c3c398dddeeca17e225a37e42f7f7033b